tasks-by-users/Rehan-Khan-Engineer/Bikes-webpage/Index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>RK Bikes</title>
+        <link rel="stylesheet" type="text/css" href="Rehan.css">
+    </head>
+
+        <body>
+<div id="wrapper">
+        <nav id="main-navigation">
```

Comment:
use class instead of id @Rehan-Khan-Engineer

Reply:
> use class instead of id @Rehan-Khan-Engineer

I think, it depends on the usecase here.

Reply:
Oh okay my bad @faran4engg